velox/docs/metrics.rst

```diff
@@ -124,6 +124,18 @@ Memory Management
      - Average
      - The average of total free memory capacity which is managed by the
        memory arbitrator.
+   * - memory_leak_pool_count
```

Comment:
Update this doc based on the code comment. Thanks!

Reply:
updated